### PR TITLE
Background minifollowups

### PR DESCRIPTION
**Pull request by spxiwh**
A long-time action item from my to-do list, but one that I now think is needed:

This PR adds a new section 4.1 to the HTML results pages. This section will show minifollowups of all the loudest background events. This can be useful to see which outliers are still affecting the background after applying Denty bins. Single events will usually show up numerous times here (e.g. the loudest X background events often contain a single L1 or H1 event found in multiple time slides with different events in the other ifo). I catch this on the result page and just add a snippet of html saying "Event X was found in the next Y loudest events, but I'm not going to bother showing you the same event numerous times".

This one might be a little rough, so suggestions for cleaning up the code welcome here!

### CODE REVIEW

**Comment by tdent** [comment]
@spxiwh  is the reasoning why this would give us more information than we already see from a followup of the loudest single-ifo triggers (according to whatever statistic we choose?)

**Comment by spxiwh** [comment]
I've actually found this to be very useful!

From the single trigger plots we can see the loudest events by our single-detector weightings. These are the things that will be pushing the "Denty bins" most strongly to downweight triggers in those regions.

In contrast this coincident background followup lets us know if we have cases where this is a significant outlier from the fitted distribution (especially if a single outlier appears in numerous different templates, affecting a lot of the background). Dealing with these types of events (making our statistic histogram fall off more like a Gaussian for all templates) will allow us to increase sensitivity for all templates in the bank.

In short, if I'm looking at these plots, and seeing things that are *clearly* not gravitational waves I might want to try to do something about it!

**Comment by tdent** [comment]
So this seems like a proxy for displaying both the loudest-by-new-SNR-like singles, which will be suppressing the sensitivity for bad regions of the bank, and the loudest-by-rescaled-bg-fit-stat singles, which *may* be making the coinc background bumpy/taily. 

I.e. the loudest backgrounds are expected to be just a random sampling of the loudest-by-bg-fit-stat singles .. would it then be reasonable to have two sets of 'loudest' singles?

**Comment by spxiwh** [comment]
Having more loudest singles I think can be useful (as long as we avoid information overload). That said, I would definitely still want to have this.

**Comment by spxiwh** [comment]
I've just realised that if merged without changes this will put open-box results into this section unless the relevant option is added to the config file. I probably want to protect against that!

**Comment by spxiwh** [comment]
@ahnitz I've added one more commit to here so that by default these codes will make *closed-box* plots. This will require some config file changes, I'll propose that now as well.

**Comment by spxiwh** [comment]
@ahnitz Did you want to look over the final changes I made here, or can I just merge?

File: bin/minifollowups/pycbc_foreground_minifollowup [hunk 1]
```diff
@@ -50,7 +50,12 @@ parser.add_argument('--inspiral-data-read-name',
 parser.add_argument('--inspiral-data-analyzed-name',
                     help="Name of inspiral segmentlist containing data "
                          "analyzed by each analysis job.")
-parser.add_argument('--output-map')
```
Comment (spxiwh) [inline bin/minifollowups/pycbc_foreground_minifollowup]: Eek, I accidentally removed this line! This breaks the code. I'll open a PR shortly fixing this!

**Comment by tdent** [comment]
Any idea what effect we expect this to have with the multiifo workflow?
